ui/app/templates/components/transit-key-actions.hbs

```diff
@@ -15,7 +15,34 @@
   <MessageError @errors={{errors}} />
   {{#if selectedAction}}
     <div data-test-transit-action={{selectedAction}}>
-    {{partial (concat 'components/transit-key-action/' selectedAction)}}
+      {{ component (concat 'transit-key-action/' selectedAction)
+          key=key
```

Comment:
😮

Reply:
Yeah too many properties to pass. No wonder they used partials earlier!!
The second option is to add a bunch of if/else-if statements and pass the required properties for each component. But the dynamic component approach seems cleaner.